README.md

```diff
@@ -1,47 +1,68 @@
-# Category Menu
+VTEX Category Menu
+=====
 
-Category Menu is a canonical app that any VTEX store can install.
+## Description
+The VTEX Category Menu app shows a department list of the store on an customizable menu. 
+This is a VTEX app that is used by Dreamstore product.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [1.x]    | **Maintenance LTS** |  2018-08-24     | 2018-11-20            | March 2019  | 1.x
 | [2.x]    | **Current Release** |  2018-11-20     |                       |             | 2.x
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
-
-## Continuous Integrations 
-
-### Travis CI 
-[![Build Status](https://travis-ci.org/vtex-apps/category-menu.svg?branch=master)](https://travis-ci.org/vtex-apps/category-menu)
+## Table of Contents
+- [Usage](#usage)
+- [API](#api)
```

Comment:
Use this model instead, since you are not using the API category:

```
## Table of Contents
- [Usage](#usage)
  - [Blocks API](#blocks-api)
    - [Configuration](#configuration)
  - [Styles API](#styles-api)
- [Troubleshooting](#troubleshooting)
```